draft-ietf-quic-recovery.md

```diff
@@ -365,7 +352,11 @@ receiver.
 The timer duration, or Probe Timeout (PTO), is set based on the following
 conditions:
 
-* PTO SHOULD be scheduled for max(1.5*SRTT+MaxAckDelay, kMinTLPTimeout)
+* If less than 2400 bytes are in flight, PTO SHOULD be scheduled for
```

Comment:
Having this somewhat magic number here just feels weird. This is the number of bytes 'we, locally' have in flight right? So don't we know our current sending MTU? Can't it just be two times that?

Reply:
Yes, but the receiver needs to know the current.  Should I reference kMaxDatagramSize and call that close enough?  I'm fairly concerned that if that changes over the course of the connection, the sender and receiver will not act predictably.

Reply:
So, is the receiver acknowledging based on byte count or packet count? I thought it was packet count. If so, then it seems to reason, the sender should do the same here.

Reply:
My understanding is that TCP waits for 2 full-sized packets, so I was trying to specify something similar to that in bytes.  The issue here becomes what constitutes a full-sized packet if the packet size changes?  Maybe keeping a max received packet size is reasonable?

We can add a packet count, but it's yet another variable to track and I don't think receiving two tiny packets would cause an immediate ACK in TCP, so I wouldn't expect it to in QUIC.